USERGUIDE.md

```diff
@@ -0,0 +1,168 @@
+# Requirements
```

Comment:
I think the `# MiniApp` and `## Features` sections from the readme need to be moved here. The published Github pages site needs to be able to stand on it's own as an entry point to the SDK documentation: 
https://rakutentech.github.io/ios-miniapp/

Maybe we can keep the `# MiniApp` section in both `README.md` and `USERGUIDE.md`?

Reply:
Agreed. I copied `# MiniApp` and `## Features` sections in USERGUIDE.md.

But sometimes we will end up updating in two places now. README.md and USERGUIDE.md. Guess we may need to come up with some other content for just the Readme.